src/main/java/org/codewars/utils/TaskRunner.java

```diff
@@ -46,8 +49,18 @@ public static String runStockSummary(){
         return null;
     }
 
-    public static double runGetVolumeOfCuboid(){
-        return 0.0;
+    public void runGetVolumeOfCuboid(){
+
+        System.out.println("Run task Volume of a Cuboid");
+        System.out.println("Input length of the cuboid ");
+        double length = Reader.readDouble();
+        System.out.println("Input width of the cuboid ");
+        double width = Reader.readDouble();
+        System.out.println("Input height of the cuboid");
+        double height = Reader.readDouble();
```

Comment:
Would we do checksng on negative and 0 values?

Reply:
When following the instructions on codewars.com, there is no need to verify input for negative or zero values. However, it would be prudent to ask Lubomir for guidance.